BigQuery/src/QueryResults.php

```diff
@@ -70,6 +71,8 @@ class QueryResults implements \IteratorAggregate
     /**
      * @param ConnectionInterface $connection Represents a connection to
      *        BigQuery.
+     *        @internal This object is created by BigQueryClient,
```

Comment:
this is marking the method `__construct` as internal, which I don't think is what we want. It'd be best to just omit the `@internal` tag and keep the rest of the description you have here.

Reply:
There is no support in PHPDoc for marking a parameter as internal, it exists only for classes and methods. Adding the `@internal` tag anywhere in the PHPDoc will make IDEs think the method is internal. So the best we can do is mark the constructor as `@internal`, or just explain that the parameter is internal in the PHPDoc parameter description.